### PR TITLE
Add different directory pitfall FAQ entry to readme.md

### PR DESCRIPTION
**Pull request by datajoely**
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Motivation and Context
Why was this PR created?

This PR aims to address the pitfall discussed on this [discord thread](https://discord.com/channels/778216384475693066/846330075535769601/870590055754899536) and helps address a pitfall where a hardcoded value causes issues when the project and DAG paths are different.

## How has this been tested?
What testing strategies have you used?

Tested with the user who raised the issue and worked out the problem. 

## Checklist

- [x] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Assigned myself to the PR
- [x] Added tests to cover my changes
- [x] Added new entries to the `RELEASE.md` file

### CODE REVIEW

**Comment by datajoely** [comment]
This PR aims to address the pitfall discussed on this [discord thread](https://discord.com/channels/778216384475693066/846330075535769601/870590055754899536).

File: README.md [hunk 1]
```diff
@@ -44,3 +44,9 @@ For more information about the DAGs folder, please visit [Airflow documentation]
 
 After generating and deploying the DAG file, you will then need to package and install the Kedro pipeline into the Airflow executor's environment.
 Please visit the guide to [deploy Kedro as a Python package](https://kedro.readthedocs.io/en/stable/10_deployment/02_single_machine.html#package-based) for more details.
+
+### FAQ
+
+#### What if my DAG file is in a different directory to my project folder?
+
+By default the generated DAG is configured to `project_path = Path().cwd()` as [per the template](https://github.com/quantumblacklabs/kedro-airflow/blob/master/kedro_airflow/airflow_dag_template.j2#L44) and you may need to change this path manually.
```
Comment (MerelTheisenQB) [inline README.md:52]: When you say "you may need to change this path manually" how does a user know that they will have to do this or not? Could you make this a bit more explicit by telling them exactly when they need to change it and how they do it?

Comment (datajoely) [reply README.md]: Thoughts now?

Comment (MerelTheisenQB) [reply README.md]: Yes that's a lot clearer to me 🙂

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Review by lorenabalan — APPROVED** [review]

**Pull request merged**